contents/docs/cdp/customer-io.md

```diff
@@ -19,7 +19,7 @@ You'll also need access to the relevant Customer.io account.
 
 ## Installation
 
-1. Visit the 'Apps' page in your instance of PostHog.
+1. In PostHog, click the "[Data pipeline](https://us.posthog.com/apps)" tab in the left sidebar.
```

Comment:
Shouldn't this be app.posthog.com or does it re-direct to EU with us.posthog.com?
btw this link will change to be /pipeline in the future (once I finally get to ship the new UI)

Reply:
- us.posthog.com redirects to eu.posthog.com when they are logged in there
- will update once you ship the new UI 😄